eZ/Publish/Core/Repository/UserService.php

```diff
@@ -80,7 +80,7 @@ public function __construct(RepositoryInterface $repository, Handler $userHandle
             'defaultUserPlacement' => 12,
             'userClassID' => 4,// @todo Rename this settings to swap out "Class" for "Type"
             'userGroupClassID' => 3,
-            'hashType' => User::PASSWORD_HASH_MD5_USER,
+            'hashType' => User::PASSWORD_BCRYPT,
```

Comment:
I think it's more important that it's consistent with our codebase, rather than PHP. I don't have to know how the constant in PHP is named, it can be named `COLOR_BLUE`, for all I care, since I can always look it up :)

If `User::PASSWORD_BCRYPT` remains, it looks out of the place and unrelated to other constants, not to mention that it hinders greping through the code base.